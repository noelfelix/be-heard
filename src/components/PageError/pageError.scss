@import 'styles/index';

.page-error {
  display: flex;
  justify-content: center;
  font-size: $sub-header-font-size;
  color: $error;
  padding: $spacing-unit-x10;
  align-items: center;

  span {
    display: block;
    padding: $spacing-unit;
  }
}