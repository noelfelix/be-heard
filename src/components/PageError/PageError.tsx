import React, { FC } from 'react'
import { rootContext } from '../../const/globals'

import './pageError.scss'

const rootClass = 'page-error'

export const PageError: FC = () => {
  return (
    <div className={rootClass}>
      <img src={`${rootContext}/images/16warning.gif`} />
      <span>OOPS - An unexpected error as occurred.</span>
    </div>
  )
}

export default PageError
