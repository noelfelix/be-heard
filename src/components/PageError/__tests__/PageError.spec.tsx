import React from 'react'
import { shallow } from 'enzyme'
import PageError from '../PageError'

describe('PageError', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<PageError />)

    expect(wrapper.debug()).toMatchSnapshot()
  })
})
