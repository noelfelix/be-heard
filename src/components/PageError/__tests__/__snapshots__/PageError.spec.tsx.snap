// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageError should render correctly 1`] = `
"<div className=\\"page-error\\">
  <img src=\\"/app/images/16warning.gif\\" />
  <span>
    OOPS - An unexpected error as occurred.
  </span>
</div>"
`;
